### PR TITLE
[MINOR] Fix spelling mistake in comment

### PR DESCRIPTION
**Pull request by tedyu**
### What changes were proposed in this pull request?
This PR fixes a typo in the comment for `InsertMapSortInGroupingExpressions`.

### Why are the changes needed?
The change helps other developers in understanding the code.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Existing test suite

### Was this patch authored or co-authored using generative AI tooling?
No

### CODE REVIEW

**Review by allisonwang-db — APPROVED** [review]

**Review by MaxGekk — COMMENTED** [review]
@tedyu As you are here, could you fix other similar places like:
- https://github.com/apache/spark/blob/f394cd3878bd38cfe6fbe2fc7ac01ada853f6375/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L4781
- https://github.com/apache/spark/blob/f394cd3878bd38cfe6fbe2fc7ac01ada853f6375/python/pyspark/sql/functions/builtin.py#L3677

**Comment by tedyu** [comment]
@MaxGekk 
`fix other similar places`: this has been done.

**Review by MaxGekk — COMMENTED** [review]
Could you fix this too:
- https://github.com/apache/spark/blob/f7bb15c21b12e7d8ed031507b2df463aea9fdb71/sql/connect/common/src/main/protobuf/spark/connect/base.proto#L525
- https://github.com/apache/spark/blob/f7bb15c21b12e7d8ed031507b2df463aea9fdb71/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/linearRegression.scala#L358
- https://github.com/apache/spark/blob/f7bb15c21b12e7d8ed031507b2df463aea9fdb71/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/linearRegression.scala#L295

**Review by dongjoon-hyun — APPROVED** [review]

**Review by yaooqinn — APPROVED** [review]

**Comment by tedyu** [comment]
@MaxGekk 
I have handled the secon batch you pointed out.

File: python/pyspark/sql/functions/builtin.py [hunk 1]
```diff
@@ -3696,7 +3696,7 @@ def regr_avgx(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |           NULL|  NULL|
     +---------------+------+
 
-    Example 3: All paris's y values are null
+    Example 3: All pairs's y values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:3699]: Is this `pairs's` correct? Shouldn't be `pair's`?

**Review by MaxGekk — CHANGES_REQUESTED** [review]
@tedyu May I ask you to fix all places in the file `builtin.py` like:
"Example 4: Some paris's x values are null"

File: python/pyspark/sql/functions/builtin.py (hunk 1 above)
Comment (tedyu) [reply python/pyspark/sql/functions/builtin.py:3699]: Should be `pairs'`

File: python/pyspark/sql/functions/builtin.py [hunk 2]
```diff
@@ -3685,7 +3685,7 @@ def regr_avgx(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |           2.75|  2.75|
     +---------------+------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:3688]: let's be consistent:
```suggestion
    Example 2: All pairs' x values are null
```

File: python/pyspark/sql/functions/builtin.py (hunk 1 above)
Comment (MaxGekk) [reply python/pyspark/sql/functions/builtin.py:3699]: ok, let's change it then:
```suggestion
    Example 3: All pairs' y values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 3]
```diff
@@ -3765,7 +3765,7 @@ def regr_avgy(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |           1.75|  1.75|
     +---------------+------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:3768]:
```suggestion
    Example 2: All pairs' x values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 4]
```diff
@@ -3776,7 +3776,7 @@ def regr_avgy(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |           NULL|   1.0|
     +---------------+------+
 
-    Example 3: All paris's y values are null
+    Example 3: All pairs's y values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:3779]:
```suggestion
    Example 3: All pairs' y values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 5]
```diff
@@ -3845,7 +3845,7 @@ def regr_count(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |               4|       4|
     +----------------+--------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:3848]:
```suggestion
    Example 2: All pairs' x values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 6]
```diff
@@ -3856,7 +3856,7 @@ def regr_count(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |               0|       1|
     +----------------+--------+
 
-    Example 3: All paris's y values are null
+    Example 3: All pairs's y values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:3859]:
```suggestion
    Example 3: All pairs' y values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 7]
```diff
@@ -3926,7 +3926,7 @@ def regr_intercept(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |                 0.0|
     +--------------------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:3929]:
```suggestion
    Example 2: All pairs' x values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 8]
```diff
@@ -3937,7 +3937,7 @@ def regr_intercept(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |                NULL|
     +--------------------+
 
-    Example 3: All paris's y values are null
+    Example 3: All pairs's y values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:3940]:
```suggestion
    Example 3: All pairs' y values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 9]
```diff
@@ -4006,7 +4006,7 @@ def regr_r2(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |          1.0|
     +-------------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:4009]:
```suggestion
    Example 2: All pairs' x values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 10]
```diff
@@ -4017,7 +4017,7 @@ def regr_r2(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |         NULL|
     +-------------+
 
-    Example 3: All paris's y values are null
+    Example 3: All pairs's y values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:4020]:
```suggestion
    Example 3: All pairs' y values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 11]
```diff
@@ -4086,7 +4086,7 @@ def regr_slope(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |             1.0|
     +----------------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:4089]:
```suggestion
    Example 2: All pairs' x values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 12]
```diff
@@ -4097,7 +4097,7 @@ def regr_slope(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |            NULL|
     +----------------+
 
-    Example 3: All paris's y values are null
+    Example 3: All pairs's y values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:4100]:
```suggestion
    Example 3: All pairs' y values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 13]
```diff
@@ -4166,7 +4166,7 @@ def regr_sxx(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |           5.0|
     +--------------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:4169]:
```suggestion
    Example 2: All pairs' x values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 14]
```diff
@@ -4177,7 +4177,7 @@ def regr_sxx(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |          NULL|
     +--------------+
 
-    Example 3: All paris's y values are null
+    Example 3: All pairs's y values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:4180]:
```suggestion
    Example 3: All pairs' y values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 15]
```diff
@@ -4246,7 +4246,7 @@ def regr_sxy(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |           5.0|
     +--------------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:4249]:
```suggestion
    Example 2: All pairs' x values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 16]
```diff
@@ -4257,7 +4257,7 @@ def regr_sxy(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |          NULL|
     +--------------+
 
-    Example 3: All paris's y values are null
+    Example 3: All pairs's y values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:4260]:
```suggestion
    Example 3: All pairs' y values are null
```

File: python/pyspark/sql/functions/builtin.py [hunk 17]
```diff
@@ -4326,7 +4326,7 @@ def regr_syy(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |           5.0|
     +--------------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```
Comment (MaxGekk) [inline python/pyspark/sql/functions/builtin.py:4329]:
```suggestion
    Example 2: All pairs' x values are null
```

**Review by MaxGekk — CHANGES_REQUESTED** [review]

**Comment by tedyu** [comment]
I have modified python/pyspark/sql/functions/builtin.py

**Review by MaxGekk — APPROVED** [review]
Waiting for CI.

**Comment by tedyu** [comment]
@MaxGekk 
CI passed.

**Pull request closed without merging**